src/realtime/NWebsocketCppRest.cpp

```diff
@@ -28,8 +28,7 @@
 namespace Nakama {
 
 NWebsocketCppRest::NWebsocketCppRest():
-_lastSentPingTimeMs(0),
-_lastReceivedPongTimeMs(0)
+_lastReceivedMessageTimeMs(0)
```

Comment:
in C++11 you can initialize class members in header

Reply:
Should we? Here it was hidden

Reply:
for `atomic` no initialization needed: https://en.cppreference.com/w/cpp/atomic/atomic/atomic

Reply:
we have to let it here as this is an atomic that can't be initialize in the header